pkg/cmd/extension/manager.go

```diff
@@ -194,10 +195,24 @@ func (m *Manager) populateLatestVersions(exts []*Extension) {
 func (m *Manager) InstallLocal(dir string) error {
 	name := filepath.Base(dir)
 	targetLink := filepath.Join(m.installDir(), name)
+	cs := m.io.ColorScheme()
+
 	if err := os.MkdirAll(filepath.Dir(targetLink), 0755); err != nil {
 		return err
 	}
-	return makeSymlink(dir, targetLink)
+	if err := makeSymlink(dir, targetLink); err != nil {
+		return err
+	}
+
+	// Check if an executable of the same name exists in the target directory.
+	// An error here doesn't indicate a failed extension installation, but
+	// it does indicate that the user will not be able to run the extension until
+	// the executable file is built or created manually somehow.
+	if _, err := os.Stat(filepath.Join(dir, name)); err != nil {
```

Comment:
This isn't the only reason that `err` might occur, so if we want to be specific, we should check [`os.IsNotExist`](https://pkg.go.dev/os#IsNotExist).

Reply:
Done in b5f3463b7530f4e1286a5bb3b5e0c4c4eb22309b